packages/admin-ui-core/src/tokens/border.ts

```diff
@@ -22,7 +22,10 @@ export const border = {
     criticalHover: borderStyles('red50'),
     criticalFocus: borderStyles('red60'),
 
-    mainChecked: borderStyles('blue40'),
-    mainCheckedHover: borderStyles('blue50'),
+    control: borderStyles('gray30'),
+    controlHover: borderStyles('gray50'),
+    controlChecked: borderStyles('blue40'),
+    controlFocus: borderStyles('gray60'),
```

Comment:
Why this is not on fg and bg (like on the buttons, for ex)?

Reply:
Actually, this token shouldn't exist. The way we apply focus on buttons and controls is by using the focus ring (`focusVisible` function)

I added this by mistake, I'll remove it